lib/cn/cn_cursor.h

```diff
@@ -21,60 +26,84 @@ struct kvs_ktuple;
 struct kvs_kvtuple;
 struct cursor_summary;
 
+#define NUM_LEVELS (2)
```

Comment:
I feel like this should be a constant in cn_tree_internal.h or something.

Reply:
And/or prefix with something like "CURSOR_CN_BINHEAP_"